app/src/main/res/values/strings.xml

```diff
@@ -14,10 +14,10 @@
     <string name="txt_museum_info">Enter museum info</string>
     <string name="txt_museum_name">Enter museum name</string>
     <string name="txt_get_museum_location">Get location</string>
-    <string name="txt_ok">OK</string>
     <string name="txt_create_instructions">Touch the screen and mark the corners of the room. When you finish press done button.</string>
     <string name="txt_info">Info</string>
     <string name="txt_building_location">Get your building location</string>
-    <string name="txt_add_door">Door ON</string>
+    <string name="txt_add_door">DOOR ON</string>
```

Comment:
This should not be ALL_CAPS. If you want some text to show as all caps - define in the textview as android:textAllCaps